rcongui/src/components/LogsView/logs.js

```diff
@@ -175,6 +183,7 @@ class Logs extends React.Component {
                 />
               )}
             />
+            <Padlock handleChange={() => this.setActionsFilterIn()} checked={actionsFilterInEnabled} label="Filter out/in"/>
```

Comment:
The label might be confusing to users why not a Select with "inclusive/exclusive" as options, you can keep true/false as values so it's compatible with the rest

Reply:
It does look much better as a select control, changes made.